### ISSUE TITLE
Setting up Alfis on Windows 11 as default DNS server

### ISSUE THREAD

**Issue by arunk** [Revertron/Alfis#371]
Hi, first of all great project! I've been waiting for something like this since starting to follow a project called namecoin some 12 years ago. So my question isn't strictly about this project, but I have asked this question on various forums and discord channels without much luck so I thought I would ask here to see if anyone has any thoughts. So I have installed alfis on my Windows 11 Pro laptop and have setup the command line DNS server to run with the -n option. I have bound it to run on 127.0.0.1 port 53 in the config file. I have then added the DNS entry for 127.0.0.1 in my WiFi network adapter, I have made this entry the first entry. Now when I do a nslookup specifying the DNS server to use, it works fine. So if I do `nslookup something.ygg 127.0.0.1` it works great and I get an IPv6 address. But if I do a `nslookup something.ygg` I get the following message:
```
Server:  UnKnown
Address:  fe80::1

*** UnKnown can't find something.ygg: Non-existent domain
```
I'm not sure what that fe80::1 DNS server is. I have nothing in my configuration that has that value for DNS. I have tried looking up this issue also but with no luck. I have also asked on Microsoft forum sites, but with no responses so far. So if anyone has had any luck getting alfis to work on Windows, specifically Windows 11, please let me know how to do it. 

Thanks.

**Comment by r4sas** [comment]
fe80::1 is usually you home router, when link-local IPv6 address advertised in local network.

**Comment by arunk** [comment]
yes I got that, I looked up what was running there using `netstat` and it turns out to be mDNSResponder, which is Bonjour service by Apple I believe. But I don't know how to tell my network adapter to use 127.0.0.1 rather than fe80::1

**Comment by r4sas** [comment]
Just set manual DNS for IPv6 and leave it empty.
